pkg/compiler/codegen.go

```diff
@@ -1569,15 +1574,81 @@ func getJumpForToken(tok token.Token, typ types.Type) (opcode.Opcode, bool) {
 	return 0, false
 }
 
+func (c *codegen) getCallToken(hash util.Uint160, method string, pcount int, hasReturn bool, flag callflag.CallFlag) uint16 {
+	needed := nef.MethodToken{
+		Hash:       hash,
+		Method:     method,
+		ParamCount: uint16(pcount),
+		HasReturn:  hasReturn,
+		CallFlag:   flag,
+	}
+	for i := range c.callTokens {
+		if c.callTokens[i] == needed {
+			return uint16(i)
+		}
+	}
+	if len(c.callTokens) == math.MaxUint16 {
+		panic("call token overflow")
```

Comment:
Are you sure that panic is a good option here? Maybe we can just return an error to give a hint to the user who writes smart-contract. The user then can reduce the number of used native calls.

Or we may add a note to the compiler documentation and write that all native interops use CALLT under the hood.

Reply:
Fixed.